packages/bot/src/services/api/contracts-for.js

```diff
@@ -266,7 +294,12 @@ export default class ContractsFor {
                         durations.push({
                             display: default_duration[0],
                             unit   : default_duration[1],
-                            min    : (index === 0 ? parseInt(contract.min_contract_duration.replace(/\D/g, '')) : 1),
+                            min    : (
+                                index === 0 ?
+                                    parseInt(contract.min_contract_duration.replace(/\D/g, ''))
+                                    : 1
+                            ),
+                            max: convertDuration(contract.max_contract_duration, default_duration[1]),
```

Comment:
@sam-binary here i think its better to show the validation error in the unit user selected. this is kindda vague. is 600 minute here?
![image](https://user-images.githubusercontent.com/3080372/69110536-0c9a8400-0ab6-11ea-997f-eb2fc1d9a0dc.png)

Reply:
1) showing a error message with unit in duration input itself will mislead user to type unit in the input.
2) There's a `Duration type` dropdown on the left, so i think is quite obvious the 600 is referring to minutes 😅